Docs: add alternative build method for mixins (#1718)
grafana/mimir: docs/sources/operators-guide/visualizing-metrics/installing-dashboards-and-alerts.md

   ```bash
   git clone https://github.com/grafana/mimir.git
   ```

> 2. Review the mixin configuration at `operations/mimir-mixin/config.libsonnet` and apply your changes if required

Reviewer: ```suggestion
2. Review the mixin configuration at `operations/mimir-mixin/config.libsonnet`, and apply your changes if necessary.
```

Reviewer: This is a preference to match voice and tone, and to reduce the complexity of past participles.

> 3. Install dependencies

Reviewer: ```suggestion
3. Install dependencies:
```

6. Install the recording rules saved at `operations/mimir-mixin-compiled/rules.yaml` in your Prometheus
7. Install the alerts saved at `operations/mimir-mixin-compiled/alerts.yaml` in your Prometheus

> Alternatively, the mixin can be compile in a container. To do so, replace the previous steps 3) and 4) by the following command:
>   ```bash
>   make BUILD_IN_CONTAINER=true build-mixin
>   ```

Reviewer: ```suggestion
Alternatively, you can compile the mixin in a container. To do so, replace the previous two steps with the following command:
  ```bash
  make BUILD_IN_CONTAINER=true build-mixin
  ```
```

Reviewer: Prefer active over passive voice.
